### DISCUSSION TITLE
Delayed Response for GitHub Student Pack Registration and Non-Expiring Student ID Card

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#52431] [GitHub Education] [answered]
Hello, I have registered for the student pack on GitHub, but I haven't received any response for a long time. What should I do? Also, my student ID card does not have an expiration date.

**Comment by samus-aran** [comment, marked as answer]
Hi @alidehkhodaei 👋 

You can check the status of your application [here](https://education.github.com/discount_requests/pack_application). If your application is still pending review then it will appear at the top of the sidebar to the right of the page. If your application has already been approved then you will be redirected to your GitHub Education benefits.

We aim to provide an outcome to your application submission within a reasonable time. This can take up to 3-4 days, however this may be extended during busy periods. We appreciate your patience and understanding.

Hope this helps! Let us know how you get along 👍

**Reply by alidehkhodaei** [reply]
![IMG_20230411_214118](https://user-images.githubusercontent.com/70447283/231252602-b8328686-43fe-4ed8-b3bb-733f13e3e199.jpg)
It show this page for me. What is mean this?

**Reply by samus-aran** [reply]
Looks like the latest submission you made was unsuccessful. It's asking you to supply additional information.

**Reply by alidehkhodaei** [reply]
Thank you for your answer😊

**Reply by AmineZCS** [reply]
@samus-aran @alidehkhodaei 
hello , my student id also doesn't have any expiration date or enrollment details, is this okay ?
also , it is written in a foreign language.

**Reply by alidehkhodaei** [reply]
@samus-aran Hello, I didn't succeed in receiving this pack and I gave up. I had the same conditions as you.

**Comment by aish1496** [comment]
I got my approval as a student and it's been 3 days, but didn't get my student benefits. When can I expect to get them.

**Reply by AmineZCS** [reply]
you were supposed to get them immediately after approval, how did you know that you got the approval ?

**Reply by palakorn13** [reply]
Me too. it displays this message " Congrats! Your request for [@palakorn13] was approved and your benefits will be available soon.". It's been 2 days for me, but I didn't get my student benefits too.

**Reply by FranciscoSotoU** [reply]
same here! Any news?

**Reply by jalandhar04** [reply]
Same problem! It displays this message "Congrats! Your request for [@jalandhar04](https://github.com/jalandhar04) was approved and your benefits will be available soon." It's been 2 days (verified as a student on Jul 7, 2023) for me, but i didn't get any benefits.

**Reply by FranciscoSotoU** [reply]
After 3 days the benefits became available to me!

**Reply by emmanuel-mutua** [reply]
How long does it take for the benefits to come to my GitHub

**Reply by sauravtibrewal** [reply]
Its been 3 days for me after getting approved. still waiting to get benefits.

**Reply by Raeesline** [reply]
you got student benefits? 4 days ago still no
